contracts/rust/src/lib.rs

```diff
@@ -225,42 +214,46 @@ mod tests {
         expected = r#"Access does not exist."#
     )]
     fn revoke_access_and_panic() {
-        let context = get_context("robert.testnet".to_string());
+        let context = get_context(robert(), 0);
         testing_env!(context);
-        let mut contract = NonFungibleTokenBasic::new("robert.testnet".to_string());
-        contract.revoke_access("kevin.testnet".to_string());
+        let mut contract = NonFungibleTokenBasic::new(robert());
+        contract.revoke_access(joe());
     }
 
     #[test]
     fn add_revoke_access_and_check() {
         // Joe grants access to Robert
-        testing_env!(get_context("joe.testnet".to_string()));
-        let mut contract = NonFungibleTokenBasic::new("joe.testnet".to_string());
-        contract.grant_access("robert.testnet".to_string());
+        let mut context = get_context(joe(), 0);
+        testing_env!(context);
+        let mut contract = NonFungibleTokenBasic::new(joe());
+        contract.grant_access(robert());
 
         // does Robert have access to Joe's account? Yes.
-        testing_env!(get_context("robert.testnet".to_string()));
-        let mut robert_has_access = contract.check_access("joe.testnet".to_string());
+        context = get_context(robert(), env::storage_usage());
```

Comment:
What do you think of using env::storage_usage() everywhere, even for single context tests? It adds a tiny amount of complexity, but also will work correctly for mutliple-context tests if copy pasted. 
I lean towards valuing the 2nd one slightly more since it's so hard to troubleshoot if you happen to run into the problem as an unsuspecting developer.

Reply:
yeah, let's use it as often as we can! I think that's what I've done here. I believe the first time you use it it'll need a starting value right? So I'm starting with 0. It needs an initial value

Reply:
I just tried removing the `0` and it breaks immediately, so we can't do that turns out :)